dockerfiles/examples/ml/README.md

```diff
@@ -0,0 +1,53 @@
+# HPCC Systems Machine Learning Docker Images
```

Comment:
Now that we are building these from buildall, I'm not sure that 'examples' is the right place for the files...

Reply:
Move ml from dockerfile/examples/ to dockerfiles/ and update the ml path inside buildall.sh script.